### ISSUE TITLE
Save and Load - doors

### ISSUE THREAD

**Issue by Godeken** [naelstrof/KoboldKare#208]
Save and Load puts Star doors back:
![image](https://user-images.githubusercontent.com/121225143/225148941-99dd895a-bca2-4da2-97d3-11663183d815.png)

If you consume or destroy front doors - after save and load they also return.

**Comment by naelstrof** [comment]
Yeah I've known this one for a while, it's not game breaking so I keep putting it on the back burner haha.
